doc/modules/decomposition.rst

```diff
@@ -864,16 +864,47 @@ The graphical model of LDA is a three-level Bayesian model:
 .. image:: ../images/lda_model_graph.png
    :align: center
 
+
+Note on notations presented in the graphical model above:
+
+  * The corpus is a collection of :math:`D` documents.
+  * A document is a sequence of :math:`N` words.
+  * There are :math:`K` topics in the corpus. 
+  * The boxes represent repeated sampling. 
+
+
+The three levels of the Bayesian model are: 
+
+  1. Generation of global hyperparameters :math:`\alpha` and :math:`\eta`, 
+     and global parameter :math:`\beta`.
+
```

Comment:
why the blank lines?

Reply:
removed this section as per the above comment